test/test_helper.rb

```diff
@@ -8,6 +8,7 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
 require 'minitest/reporters'
+require 'minitest/autorun'
```

Comment:
@colinxfleming that's the require that gets you Minitest::Mock and the mock/stubbing capabilities. Was following from https://semaphoreci.com/community/tutorials/mocking-in-ruby-with-minitest